agent/directory_test.go

```diff
@@ -100,3 +103,41 @@ func TestDirectoryCleanup(t *testing.T) {
 	_, err = os.Stat(shouldNotDelete)
 	assert.False(os.IsNotExist(err))
 }
+
+func TestCheckDataDirectoryHealthWithUsage(t *testing.T) {
+	t.Run("DiskUsageBelowThreshold", func(t *testing.T) {
+		mockComm := &client.Mock{}
+		agent := Agent{
+			opts: Options{
+				HostID: "test-host-id",
+			},
+			comm: mockComm,
+		}
+
+		// Test with low disk usage
```

Comment:
This comment + the one for the test below seem like they're redundant since the test name already describes the same thing.